client/blocks/calendar-popover/README.md

```diff
@@ -6,26 +6,21 @@ Beyond combining Popover and PostSchedule, this component connects with the stat
 
 ## Usage
 
-```es6
+```jsx
 import { Button } from '@automattic/components';
 import CalendarPopover from 'blocks/calendar-popover';
 
-toggle = () => this.setState( { show: ! this.state.show } );
+const toggle = () => this.setState( { show: ! this.state.show } );
+let buttonRef;
```

Comment:
Or maybe `React.createRef()` would be more semantic?

Reply:
Fixed